### PR TITLE
Add test of spec change for Response.created() when using relative url

### LINKED ISSUE #23657: Add relative URI FAT to cover missing TCK for EE10
The following was included in the changes for the 3.1 Restful WS specification but not included in any TCK test.  Therefore a FAT should be create to cover this.

`Response.created(URI) now resolves relative URIs into an absolute URI against the base URI, not against the
request URI anymore.`

### PR DESCRIPTION
**Pull request by jim-krueger**
Fixes #23657 

The Jakarta Rest 3.0 spec changed for when a relative URL is used in Response.created().  The resulting URI should be based on the Base URI and not the Request URI as  before EE9.   This change tests that.

In addition this PR contains a prototype testcase for the io.openliberty.restfulWS.3.0_fat.

### CODE REVIEW

**Comment by jim-krueger** [comment]
#run-libby-bot

File: dev/io.openliberty.restfulWS.3.0_fat/fat/src/io/openliberty/restfulWS30/fat/RestfulWsPrototypeTest.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
```
Comment (WhiteCat22) [inline dev/io.openliberty.restfulWS.3.0_fat/fat/src/io/openliberty/restfulWS30/fat/RestfulWsPrototypeTest.java]: 2022

File: dev/io.openliberty.restfulWS.3.0_fat/fat/src/io/openliberty/restfulWS30/fat/ResponseRelativePathTest.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
```
Comment (WhiteCat22) [inline dev/io.openliberty.restfulWS.3.0_fat/fat/src/io/openliberty/restfulWS30/fat/ResponseRelativePathTest.java]: 2022

File: dev/io.openliberty.restfulWS.3.0_fat/fat/src/io/openliberty/restfulWS30/fat/restfulwsprototype/PrototypeClientTestServlet.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import java.net.URI;
+
+import org.junit.Test;
+
+import componenttest.app.FATServlet;
+import jakarta.servlet.annotation.WebServlet;
+
+@SuppressWarnings("serial")
+@WebServlet("/PrototypeClientTestServlet")
+public class PrototypeClientTestServlet extends FATServlet {
```
Comment (WhiteCat22) [inline dev/io.openliberty.restfulWS.3.0_fat/fat/src/io/openliberty/restfulWS30/fat/restfulwsprototype/PrototypeClientTestServlet.java]: RestfulWsPrototypeClientTestServlet

File: dev/io.openliberty.restfulWS.3.0_fat/fat/src/io/openliberty/restfulWS30/fat/restfulwsprototype/Resource.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+package io.openliberty.restfulWS30.fat.restfulwsprototype;
+
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+
+@Path("/")
+@Produces(MediaType.TEXT_PLAIN)
+public class Resource {
```
Comment (WhiteCat22) [inline dev/io.openliberty.restfulWS.3.0_fat/fat/src/io/openliberty/restfulWS30/fat/restfulwsprototype/Resource.java]: RestfulWsPrototypeResource

File: dev/io.openliberty.restfulWS.3.0_fat/fat/src/io/openliberty/restfulWS30/fat/restfulwsprototype/App.java [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.restfulWS30.fat.restfulwsprototype;
+
+import jakarta.ws.rs.ApplicationPath;
+import jakarta.ws.rs.core.Application;
+
+@ApplicationPath("/")
+public class App extends Application {
```
Comment (WhiteCat22) [inline dev/io.openliberty.restfulWS.3.0_fat/fat/src/io/openliberty/restfulWS30/fat/restfulwsprototype/App.java]: RestfulWsPrototypeApplication

**Review by WhiteCat22 — CHANGES_REQUESTED** [review]

**Comment by jim-krueger** [comment]
#run-libby-bot

**Review by WhiteCat22 — APPROVED** [review]

**Comment by jim-krueger** [comment]
#build

**Pull request merged**